src/main/java/be/acara/events/service/UserServiceImpl.java

```diff
@@ -14,11 +18,13 @@
 public class UserServiceImpl implements UserService {
 
     private final UserRepository userRepository;
+    private final EventRepository eventRepository;
     private final RoleRepository roleRepository;
 
     @Autowired
-    public UserServiceImpl(UserRepository userRepository, RoleRepository roleRepository) {
+    public UserServiceImpl(UserRepository userRepository, EventRepository eventRepository, RoleRepository roleRepository) {
         this.userRepository = userRepository;
+        this.eventRepository = eventRepository;
         this.roleRepository = roleRepository;
     }
```

Comment:
Replace with EventService.

https://www.baeldung.com/circular-dependencies-in-spring

#4.3. Use Setter/Field Injection